tensorflow/core/kernels/mkl/mkl_fused_ops_test.cc

```diff
@@ -715,35 +757,56 @@ class FusedPadConvOpTest : public OpsTestBase {
          59, 12,  0,   0,   0,   0,  0,   0,   0,   0,   0,   0});
 
     // Create a fused pad+conv2d node
-    TF_EXPECT_OK(NodeDefBuilder("fused_pad_conv_op", "_MklPadWithConv2D")
-                     .Input(FakeInput(dtype))     // Input
-                     .Input(FakeInput(dtype))     // Filter
-                     .Input(FakeInput(DT_INT32))  // Padding
-                     .Input(FakeInput(DT_UINT8))  // MKl second tensor
-                     .Input(FakeInput(DT_UINT8))  // MKl second tensor
-                     .Input(FakeInput(DT_UINT8))  // MKl second tensor
-                     .Attr("padding", "VALID")
-                     .Attr("data_format", data_format)
-                     .Attr("T", dtype)
-                     .Attr("strides", {1, stride, stride, 1})
-                     .Attr("_kernel", "MklLayoutDependentOp")
-                     .Finalize(node_def()));
-    TF_EXPECT_OK(InitOp());
+    if (!NativeFormatEnabled()) {
+      TF_EXPECT_OK(NodeDefBuilder("fused_pad_conv_op", "_MklPadWithConv2D")
+                       .Input(FakeInput(dtype))     // Input
+                       .Input(FakeInput(dtype))     // Filter
+                       .Input(FakeInput(DT_INT32))  // Padding
+                       .Input(FakeInput(DT_UINT8))  // MKl second tensor
+                       .Input(FakeInput(DT_UINT8))  // MKl second tensor
+                       .Input(FakeInput(DT_UINT8))  // MKl second tensor
+                       .Attr("padding", "VALID")
+                       .Attr("data_format", data_format)
+                       .Attr("T", dtype)
+                       .Attr("strides", {1, stride, stride, 1})
+                       .Attr("_kernel", "MklLayoutDependentOp")
+                       .Finalize(node_def()));
+      TF_EXPECT_OK(InitOp());
+    } else {
+      TF_EXPECT_OK(
+          NodeDefBuilder("fused_pad_conv_op", "_MklNativePadWithConv2D")
+              .Input(FakeInput(dtype))     // Input
+              .Input(FakeInput(dtype))     // Filter
+              .Input(FakeInput(DT_INT32))  // Padding
+              .Attr("padding", "VALID")
+              .Attr("data_format", data_format)
+              .Attr("T", dtype)
+              .Attr("strides", {1, stride, stride, 1})
+              .Attr("_kernel", "MklNameChangeOp")
+              .Finalize(node_def()));
+      TF_EXPECT_OK(InitOp());
```

Comment:
Nit: Please refactor this line out of `if/else`.

Reply:
Done.